### PR TITLE
Blobs

### PR DESCRIPTION
**Pull request by dessaya**
* Added `listBlobs` view to `blob` SC
* `wasp-cli` now can manipulate blobs
* Dashboard now shows blobs

### CODE REVIEW

File: tools/cluster/demo/tests/blob-test.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
```
Comment (kape1395) [inline tools/cluster/demo/tests/blob-test.sh:1]: Wouldn't is be better to write these tests in go? They would be run by `go test ./...`, thus easier to maintain them working?

**Review by kape1395 — COMMENTED** [review]
Just a question on tests.

File: tools/cluster/demo/tests/blob-test.sh (hunk 1 above)
Comment (dessaya) [reply tools/cluster/demo/tests/blob-test.sh:1]: I don't know, maybe? These tests require lots of string parsing in order to interpret the command output and make assertions. If they were written in Go, I estimate they would take 10x - 20x LOC and be way more cumbersome to write.

Comment (kape1395) [reply tools/cluster/demo/tests/blob-test.sh:1]: OK

**Review by lunfardo314 — APPROVED** [review]

**Pull request merged**